src/tests/fuzz-lxc-config-read.c

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "conf.h"
+#include "confile.h"
+#include "utils.h"
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+	int fd = -1;
+	char tmpf[] = "fuzz-lxc-config-read-XXXXXX";
+	struct lxc_conf *conf = NULL;
+
+	fd = lxc_make_tmpfile(tmpf, false);
+	lxc_write_nointr(fd, data, size);
+	close(fd);
```

Comment:
This looks like it's missing some error checking. This should check whether `lxc_make_tmpfile()` failed and error out. It should also check for `lxc_conf_init()` failure and not call `lxc_config_read()` in that case and error out instead.

Reply:
> This looks like it's missing some error checking

Agreed. Given that it isn't expected that it should fail I wonder if something like `lxc_test_assert_abort(fd >= 0);` would suffice. That's what we usually do in the `systemd` fuzz targets :-)

Reply:
Yeah, that sounds like a good idea!

Reply:
Done